### PR TITLE
Flaky: TestGameServerUnhealthyAfterReadyCrash

### PR DESCRIPTION
**Pull request by markmandel**
This should fix the flakiness with this test.

### CODE REVIEW

File: test/e2e/gameserver_test.go [hunk 1]
```diff
@@ -335,13 +335,23 @@ func TestGameServerUnhealthyAfterReadyCrash(t *testing.T) {
...
 
 	address := fmt.Sprintf("%s:%d", readyGs.Status.Address, readyGs.Status.Ports[0].Port)
-	conn, err := net.Dial("udp", address)
-	assert.NoError(t, err)
-	defer conn.Close() // nolint: errcheck
-	_, err = conn.Write([]byte("CRASH"))
-	assert.NoError(t, err)
-	l.WithField("address", address).Info("sent UDP packet")
 
+	// keep crashing, until we move to Unhealthy. Solves potential issues with controller Informaer cache
```
Comment (roberthbailey) [inline test/e2e/gameserver_test.go]: Maybe I'm missing something but how does this loop exit when the game server moves to unhealthy? It only terminates when there is an error writing a packet to the udp destination, which seems unrelated to the game server becoming unhealthy.

Comment (markmandel) [reply test/e2e/gameserver_test.go]: Yeah, I just wanted to give it an eventual exit condition. So it will stop when there is nothing left to send things to (i.e. it errors out)
